components/profile/dashboard/monthly-contribution.js

```diff
@@ -0,0 +1,55 @@
+import { Avatar, Box, Card, CardContent, Grid, Typography } from '@mui/material';
+import CalendarTodayIcon from '@mui/icons-material/CalendarToday';
+import ChangeArrow from '../commons/change-arrow';
+
+const MonthlyContribution = (props) => {
+  const {cPLastMonth, cPThisMonth} = props;
+  return (
+    <Card
+      sx={{ height: '100%' }}
+      {...props}
+    >
+      <CardContent>
+        <Grid
+          container
+          spacing={3}
+          sx={{ justifyContent: 'space-between' }}
+        >
+          <Grid item>
+            <Typography
+              color="textSecondary"
+              gutterBottom
+              variant="overline"
+            >
+              MONTHLY CONTRIBUTION 
+            </Typography>
+            <Typography
+              color="textPrimary"
+              variant="h4"
+            >
+              {cPThisMonth} Pts
+            </Typography>
+          </Grid>
+          {/* <Grid item>
+            <Avatar
+              sx={{
+                backgroundColor: 'info.main',
+                height: 56,
+                width: 56
+              }}
+            >
+              <CalendarTodayIcon />
+            </Avatar>
+          </Grid> */}
```

Comment:
Is this section use for display an icon.
If (we need to use icon here) change to correct icon
else delete this section of code ✌

Reply:
The icon actually fits with the content here I think. Otherwise, let me know if you have some other ideas.